applications/dashboard/models/class.usermodel.php

```diff
@@ -657,6 +657,11 @@ public function sso($String, $ThrowError = false) {
 
         $Provider = Gdn_AuthenticationProviderModel::getProviderByKey($ClientID);
 
+        if (!filter_var($Timestamp, FILTER_VALIDATE_INT) || abs($Timestamp - time()) > 20 * 60) {
```

Comment:
How about making the result of `20 * 60` into a class constant or something similar?